tests/test_lldpSyncDaemon.py

```diff
@@ -74,17 +85,17 @@ def test_sync_roundtrip(self):
         for k in keys:
             # The test case is for LLDP neighbor information.
             # Need to filter LLDP_LOC_CHASSIS entry because the entry is removed from parsed_update after executing daemon.sync().
-            if k != 'LLDP_LOC_CHASSIS':
+            if k.decode() != 'LLDP_LOC_CHASSIS':
```

Comment:
Is this necessary? swsssdk should now return strings in both Python 2 and 3. @qiluo-msft may be able to comment further.

Reply:
Yes. It is supported by SonicV2Connector constructor option `decode_responses=True` which is the default.

---
In reply to: [526518457](https://github.com/Azure/sonic-dbsyncd/pull/32#discussion_r526518457) [](ancestors = 526518457)

Reply:
Thanks @jleveque and @qiluo-msft updated accordingly.